src/Pages/ProductDetails/ComponentYH/RightSlider/Description.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+class Description extends React.Component {
+
+
+  render (){
+    return(
+      <div className='description'>
+        {console.log(this.props.description)}
+        {
+          this.props.description.split("\n").map( line => {
+            return (<div>{line}</div>)
+          })
+        }
```

Comment:
아래와 같이 작성하시면 따로 return 이 필요하지 않습니다. 
+ 구조분해할당 해주세요!!
```suggestion
        {
          this.props.description.split("\n").map( line => (
          <div>{line}</div>
          ))
        }
```